spec/ndx-ibl-metadata.extensions.yaml

```diff
@@ -60,83 +60,42 @@ groups:
   neurodata_type_inc: Subject
   name: subject
   doc: IBL mice data
-  attributes:
-  - name: alive
-    dtype: bool
-    doc: alive/dead
-    required: false
   datasets:
-  - name: nickname
+  - name: uuid
     dtype: text
-    doc: name of the mouse
+    doc: The full identifier of the subject from the IBL database.
     quantity: '?'
   - name: url
     dtype: text
-    doc: url location of the subject metadata
+    doc: Extra information about the subject can be found at this URL.
     quantity: '?'
   - name: responsible_user
     dtype: text
-    doc: user in charge of the subject
-    quantity: '?'
-  - name: death_date
-    dtype: text
-    doc: date of sacrifice of the mouse
-    quantity: '?'
-  - name: litter
-    dtype: text
-    doc: litter of the mouse
-    quantity: '?'
-  - name: strain
-    dtype: text
-    doc: strain of mouse
-    quantity: '?'
-  - name: source
-    dtype: text
-    doc: source
-    quantity: '?'
-  - name: line
-    dtype: text
-    doc: line of this mouse
+    doc: User ID in charge of the subject.
     quantity: '?'
   - name: projects
     dtype: text
     shape:
     - null
-    doc: ibl project involving this mouse
+    doc: The main projects this subject was involved in.
     quantity: '?'
   - name: session_projects
     dtype: text
     shape:
     - null
-    doc: session projects
-    quantity: '?'
-  - name: lab
-    dtype: text
-    doc: lab
+    doc: All the other projects this subject was involved in.
     quantity: '?'
   - name: last_water_restriction
     dtype: text
-    doc: last water restriction
+    doc: The date of the last water restriction.
     quantity: '?'
-  - name: expected_water
+  - name: expected_water_ml
     dtype: float
-    doc: expected water
+    doc: The expected amount of water in ml.
```

Comment:
I'm recommending adding `_ml` to the end of this field name to more obviously specify the units that it is in

I know the `doc` here mentions the units, but this is particularly for the Widgets, which look like

![image](https://user-images.githubusercontent.com/51133164/218549910-d21b77b2-7e91-4f44-910d-1af5991224dc.png)

and unlike `weight`, since these are custom extension fields, the Widgets can't hard-code the unit

![image](https://user-images.githubusercontent.com/51133164/218550101-2c552f17-3fd6-4bba-b699-135661a910c1.png)